### ISSUE TITLE
Phantom progress bar windows on hover.

### ISSUE THREAD

**Issue by VladiStep** [krzys-h/UndertaleModTool#1616]
### Describe the bug

When you hover on the main window preview in the task bar, it shows all the progress bar windows that were closed.
![image](https://github.com/krzys-h/UndertaleModTool/assets/45438625/9939a37c-43ac-4d31-a69d-7ab214e0226f)

### Reproducing steps

1. Open any game.
2. Hover on the main window preview.

### Setup Details

1. UTMT version - https://github.com/krzys-h/UndertaleModTool/commit/42e097ce9125935ff7dc696e90be79e3b66039f8
2. Operating system - Windows 10 22H2 (19045.3803).

**Comment by Miepee** [comment]
This seems like a WPF bug? I rmember seeing that in other apps too at one point...

**Comment by VladiStep** [comment]
I think this is because of `HideProgressBar()`, so the progress window is hidden instead of closed.

**Comment by VladiStep** [comment]
Yup, when I changed it from `scriptDialog.TryHide()` to `scriptDialog.TryClose()`, it stopped showing those phantom windows on hover
